plugin/src/App/init.lua

```diff
@@ -123,9 +124,15 @@ function App:startSession()
 		end
 
 		if count == 0 then return end
+
+		local old = self.patchInfo:getValue()
+		if now - old.timestamp < 2 then
+            count += old.updates
```

Comment:
Weird indentation

Reply:
Wrote it on mobile, sorry! I'll fix it when I get home later tonight.